Abstractions/ResponseDto.cs

```diff
@@ -0,0 +1,3 @@
+namespace UniVerServer.Abstractions;
+
+public record ResponseDto(Guid Id, string ActionMessage, Enums.StatusCodes StatusCode);
```

Comment:
Be careful when working with GUID's as they impact performance. Have a look at [this](https://dba.stackexchange.com/questions/264/guid-vs-int-which-is-better-as-a-primary-key)

Reply:
@wiaandev  Well aware of the pros and cons of GUID, implementation was purely from a learning POV